modules/template/elements/ContainerElementVariable.php

```diff
@@ -69,49 +79,45 @@ protected function renderEditBlock(string $content): string
             $content = Html::tag('div', Yii::t('CustomPagesModule.model', 'Empty <br />Container'));
         }
 
-        if ($this->areEditWrapperAttributesRendered()) {
+        if ($this->isEditWrapperRendered()) {
+            // Don't render the wrapper twice if the attributes are rendered
+            // in a html tag like `<div {{ container.editWrapperAttributes }}>`
             return $content;
         }
 
         $tagName = preg_match('#<(tr).+?</\1>#is', $content) ? 'tbody' : 'div';
 
-        return Html::tag($tagName, $content, $this->getEditWrapperAttributes());
+        return Html::tag($tagName, $content, $this->getEditWrapperAttributesArray());
     }
 
-    protected function getEditWrapperAttributes(): array
+    protected function getEditWrapperAttributesArray(array $attributes = []): array
     {
-        $attributes = [
-            'data-editor-container-id' => $this->elementContent->id,
-        ];
+        $attributes['data-editor-container-id'] = $this->elementContent->id;
 
         if ($this->getItems() === []) {
-            $attributes['class'] = 'cp-editor-container-empty';
+            Html::addCssClass($attributes, 'cp-editor-container-empty');
         }
 
         return $attributes;
     }
 
-    public function editWrapperAttributes(): string
+    public function getEditWrapperAttributes(array $attributes = []): string
     {
         if (TemplateInstanceRendererService::inEditMode()) {
-            $rendered = $this->getRenderedEditWrapperAttributes();
-            $rendered[] = $this->elementContent->id;
-            Yii::$app->runtimeCache->set(self::EDIT_WRAPPER_ATTR_CACHE_KEY, $rendered);
-
-            return Html::renderTagAttributes($this->getEditWrapperAttributes());
+            Yii::$app->runtimeCache->set($this->getEditWrapperAttributesCacheKey(), true);
```

Comment:
Not sure, we really the `runtime` Cache here. because the variable class is instanciated only once. 

Maybe we could simply use a `private` property e.g. `private bool $editWrapperSent = false`.

Reply:
@luke- Yes, you are right, it works - https://github.com/humhub/custom-pages/pull/441/commits/389e25ee464404b3c817549dab72289a12247962.